### DISCUSSION TITLE
my sql setup error

### DISCUSSION THREAD

**Discussion by Bhashitha19** [learndevopsonline/learndevopsonline#1809] [Q&A]

**Comment by Bhashitha19** [comment]
![Screenshot (24)](https://github.com/learndevopsonline/learndevopsonline/assets/148459705/ddc58867-295f-44af-affa-99a19f996115)
sir when iam trying to add expenses it will not work

**Comment by chaithanya1812** [comment]
Hello, you have not set up the nginx configuration file to connect backend.

![image](https://github.com/learndevopsonline/learndevopsonline/assets/111736742/1fbd2ffc-224e-4ef5-9014-d83e1de16a08)

**Comment by Bhashitha19** [comment]
![Screenshot (25)](https://github.com/learndevopsonline/learndevopsonline/assets/148459705/0d2054d7-d1d8-43d9-8a50-234f7989f8f5)
![Screenshot (26)](https://github.com/learndevopsonline/learndevopsonline/assets/148459705/bb58e32c-77f7-41fb-a3a2-82cad4ee1cb0)

**Comment by chaithanya1812** [comment]
you have not add --> location /api/ { proxy_pass http://localhost:8080/; } port 8080 at end.
![image](https://github.com/learndevopsonline/learndevopsonline/assets/111736742/99dbe6d9-df56-4a32-94bf-06b15f40da13)

**Comment by Bhashitha19** [comment]
![Screenshot (27)](https://github.com/learndevopsonline/learndevopsonline/assets/148459705/468f478a-8fac-4576-81d1-753dd705463f)
still it was not working sir

**Comment by chaithanya1812** [comment]
Hello Bhashitha, Again you done a wrong step.

![image](https://github.com/learndevopsonline/learndevopsonline/assets/111736742/a2b97719-cb1f-4092-883d-47f10aa7d266)


you have configure msql ip address in proxy_pass , replace with backend ip-addres

**Comment by chaithanya1812** [comment]
Hello Bhashitha, I restarted backend once now it's working fine you can check once..
![image](https://github.com/learndevopsonline/learndevopsonline/assets/111736742/3c4cb88b-4c63-49c8-8133-0fe5d306415a)

**Reply by Bhashitha19** [reply]
tq sir

**Comment by Bhashitha19** [comment]
![Uploading Screenshot (28).png…]()

**Comment by chaithanya1812** [comment]
Hello Bhashitha, 
your image is not loaded ,can you check once..
